recipe/meta.yaml

```diff
@@ -27,7 +27,7 @@ source:
 build:
   number: 1
```

Comment:
Is it expected to keep the build number `1`?  I guess other CUDA PRs have the same issue?